trade-mapping-to-ipaffs/src/test/java/uk/gov/defra/stw/mapping/toipaffs/chedd/commodities/CheddPackageTypeMapperTest.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.defra.stw.mapping.toipaffs.chedd.commodities;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import uk.gov.defra.stw.mapping.dto.CodeType;
+import uk.gov.defra.stw.mapping.dto.PackageTypeCodeType;
+import uk.gov.defra.stw.mapping.dto.PhysicalSpsPackage;
+import uk.gov.defra.tracesx.notificationschema.representation.ComplementParameterSetKeyDataPair;
+import uk.gov.defra.tracesx.notificationschema.representation.enumeration.PackageType;
+
+class CheddPackageTypeMapperTest {
```

Comment:
Rename this test to match the class under test `CheddPackageTypeKeyDataMapperTest`